README.md

```diff
@@ -1,66 +1,114 @@
--   **Cast**: Swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
--   **Anvil**: Local Ethereum node, akin to Ganache, Hardhat Network.
--   **Chisel**: Fast, utilitarian, and verbose solidity REPL.
+In Maker's linguo, a spell is a bespoke smart contract to execute authorized actions in Maker Protocol on behalf on
+Maker Governance.
 
-## Documentation
+Since most contracts in Maker Protocol follow a [simple, battle-tested authorization scheme][auth], with an "all or
+nothing" approach, it means that every spell has _root_ access to every single of its components.
 
-https://book.getfoundry.sh/
+[auth]: https://github.com/makerdao/pe-checklists/blob/master/core/standards.md#permissions
 
-## Usage
+In order to mitigate the risks associated with that design decision, the spell process is quite "heavy", where
+multiple trusted parties are involved, and with comprehensive [checklists][spell-checklists] that must be strictly
+followed.
 
-### Build
+[spell-checklists]: https://github.com/makerdao/pe-checklists/tree/master/spell
 
-```shell
-$ forge build
-```
+With all the complexity and coordination effort, it is not a surprise that it takes a long time for a spell to be
+successfully crafted, reviewed and handed over to Maker Governance. As per the [current process][spell-schedule], with
+the involvement of at least 3 engineers from the different EAs in the Spell Team, not to mention the Governance
+Facilitators and other key stakeholders, it takes at least 8 working days to deliver a regular spell.
 
-### Test
+[spell-schedule]: https://github.com/makerdao/pe-checklists/blob/master/spell/spell-crafter-mainnet-workflow.md#spell-coordination-schedule
 
-```shell
-$ forge test
-```
+For emergency spells, historically the agreed SLA had been 24h. That was somehow possible when there was a single
+tight-knit team working on spells, however can be specially challenging with a more decentralized workforce, which is
+scattered around the world. Even if it was still possible to meet that SLA, in some situations 24h might be too much
+time.
 
-### Format
+This repository contains a couple of different spells performing emergency actions that can be pre-deployed to allow
+MakerDAO Governance to quickly respond to incidents, without the need for dedicated engineers to chime in and craft a
+bespoke spell in record time.
 
-```shell
-$ forge fmt
-```
+## Deployments
 
-### Gas Snapshots
+TBD.
 
-```shell
-$ forge snapshot
-```
+## Implemented Actions
 
-### Anvil
+| Description        | Single ilk         | Universal          |
+| :----------        | :--------:         | :-------:          |
+| Wipe `AutoLine`    | :white_check_mark: | :white_check_mark: |
+| Set `Clip` breaker | :white_check_mark: | :white_check_mark: |
+| Disable `DDM`      | :white_check_mark: | :x:                |
+| Stop `OSM`         | :white_check_mark: | :white_check_mark: |
```

Comment:
Do you that that if we aim to allow potential usage of emergency spells without involvement of engineers, we might want to use less technical, more general terms in describing them? Also, we might want to describe not what they actually do, but when to use each of them.
Anyway, I guess this will be addressed in the place where the deployed contracts will be listed in the end – with some kind of guide on how to use them.

Reply:
> Anyway, I guess this will be addressed in the place where the deployed contracts will be listed in the end – with some kind of guide on how to use them.

That's the intention. This repo's docs are more useful to a technical audience.